clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookFix.ts

```diff
@@ -32,7 +32,7 @@ export class CopybookFix {
         const actionSettings = "Change settings";
         const action = await vscode.window.showErrorMessage(title, actionSettings);
         if (action === actionSettings) {
-            vscode.commands.executeCommand("workbench.action.openSettings", "broadcom-cobol-lsp");
+            vscode.commands.executeCommand("broadcom-cobol-lsp.cpy-manager.goto-settings", "cpy-manager");
```

Comment:
You don't have params for this command. You can call it this way:
```
vscode.commands.executeCommand("broadcom-cobol-lsp.cpy-manager.goto-settings");
```

Reply:
OK I will do it ;)